templates/gallery/gallery/list.html

```diff
@@ -46,31 +47,31 @@
                     </a>
                 </div>
                 <p class="topic-answers">
-                    {% with img_count=gallery.get_images.count %}
+                    {% with img_count=gallery.image_count %}
                         {% if img_count == 0 %}
                             {% trans "Aucune image" %}
-                        {% elif img_count == 1 %}
-                            {% trans "1 image" %}
                         {% else %}
-                            {{ img_count }} {% trans "images" %}
+                            {{ img_count }} {% trans "image" %}{{ img_count|pluralize_fr }}
                         {% endif %}
                     {% endwith %}
                 </p>
 
-                {% if content %}
-                    <p class="topic-last-answer">
-                        {% trans "Liée" %}
-                        {% if content.is_article %}
-                            {% trans "à l’article" %}
-                        {% elif content.is_tutorial %}
-                            {% trans "au tutoriel" %}
-                        {% elif content.is_opinion %}
-                            {% trans "au billet" %}
-                        {% else %}
-                            {% trans "au contenu" %}
-                        {% endif %}
-                        «&NonBreakingSpace;<a href="{{ content.get_absolute_url }}">{{ content.title }}</a>&NonBreakingSpace;».
-                    </p>
+                {% if gallery.linked_content in linked_contents %}
+                    {% with content=linked_contents|get_item:gallery.linked_content %}
+                        <p class="topic-last-answer">
+                            {% trans "Liée" %}
```

Comment:
Je sais que tu n'as pas codé cette partie mais un blocktrans me semble plus judicieux

Reply:
Oh, je l'ai codé dans une itération précédente ^^

Reply:
Bon, au final, non. Il faut quand même un `captureas` pour s'en sortir, donc on gagne rien.